src/urlValidation/urlValidation.js

```diff
@@ -0,0 +1,37 @@
+function urlValidation(url) {
+  const cases = ['SNAKE', 'KEBAB', 'CAMEL', 'PASCAL', 'UPPER'];
+  const [textToConvert, searchParams] = url.split('?');
+
+  const errors = [];
+  const params = new URLSearchParams(searchParams);
+  const toCase = params.get('toCase');
+  const normilizedText = textToConvert.slice(1);
```

Comment:
```suggestion
  const normalizedText = textToConvert.slice(1);
```

Reply:
BTW - I am using this - https://marketplace.visualstudio.com/items?itemName=streetsidesoftware.code-spell-checker